thin-provisioning-tools: Add thin-provisioning-tools package.

AkitoTakada opened the pull request:
0001-do-not-strip-pdata_tools-at-do_install.patch
  It is not necessary because the Debian source is already patched.
0001-fix-compile-failed-with-libc-musl.patch
use-sh-on-path.patch
  These are necessary.

Commit:
thin-provisioning-tools: Add thin-provisioning-tools package.

0001-do-not-strip-pdata_tools-at-do_install.patch
  It is not necessary because the Debian source is already patched.
0001-fix-compile-failed-with-libc-musl.patch
use-sh-on-path.patch
  These are necessary.

AkitoTakada commented:
I have confirmed that the file exists, but I have not tested it.
The reasons for not being able to test are as follows.
There is no logical volume in the emulator.
Also a logical volume cannot be created because there is no physical volume.
Therefore, it cannot be tested.

All of libaio, thin-provisioning-tools, and lvm2 are in the same situation.

```
root@qemuarm64:~# find /usr/sbin -name "*cache_*" | sort -n
/usr/sbin/cache_check
/usr/sbin/cache_dump
/usr/sbin/cache_metadata_size
/usr/sbin/cache_repair
/usr/sbin/cache_restore
/usr/sbin/cache_writeback
root@qemuarm64:~# find /usr/sbin -name "*era_*" | sort -n
/usr/sbin/era_check
/usr/sbin/era_dump
/usr/sbin/era_invalidate
/usr/sbin/era_restore
root@qemuarm64:~# find /usr/sbin -name "pdata_tools" | sort -n
/usr/sbin/pdata_tools
root@qemuarm64:~# find /usr/sbin -name "thin_*" | sort -n
/usr/sbin/thin_check
/usr/sbin/thin_delta
/usr/sbin/thin_dump
/usr/sbin/thin_ls
/usr/sbin/thin_metadata_size
/usr/sbin/thin_repair
/usr/sbin/thin_restore
/usr/sbin/thin_rmap
/usr/sbin/thin_trim
```

masami256 commented on recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb:
```diff
@@ -0,0 +1,29 @@
+           file://use-sh-on-path.patch \
+"
+
+UPSTREAM_CHECK_GITTAGREGEX = "v(?P<pver>\d+(\.\d+)+)"
```
@AkitoTakada  This recipe doesn't checkout source code from git, can be removed.

Commit:
thin-provisioning-tools: Fixed what was pointed out in the pull request.

Contents pointed out
  This recipe doesn't checkout source code from git, can be removed.

AkitoTakada commented:
Deleted as pointed out.

yuwata commented:
@masami256 Also in this case, please use 'squash' merge.

masami256 commented:
@AkitoTakada Would you merge 3bd80c43b29f4d5ddb27a7def90c69b0913c4eaf and b43983edf071dd26ff8ba8d08699942201856e03 ? It creates simple commit for add a package.

masami256 commented:
@AkitoTakada commit b43983edf071dd26ff8ba8d08699942201856e03 removes use-sh-on-path.patch from SRC_URI, is it okay to remove the patch ?

hiraku-wfs commented on recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb:
```diff
@@ -0,0 +1,26 @@
+# base recipe: https://github.com/openembedded/meta-openembedded/tree/warrior/meta-oe/recipes-support/thin-provisioning-tools/thin-provisioning-tools_0.7.6.bb
```
Could you change this line to the following?
`# base recipe: meta-oe/recipes-support/thin-provisioning-tools/thin-provisioning-tools_0.7.6.bb`

Commit:
thin-provisioning-tools: Fixed what was pointed out in the pull request(second time).

Contents pointed out
  removes use-sh-on-path.patch from SRC_URI, is it okay to remove the patch ?
  Could you change this line to the following?
    base recipe: meta-oe/recipes-support/thin-provisioning-tools/thin-provisioning-tools_0.7.6.bb

AkitoTakada commented:
I misunderstood the indication.
The patch has been returned.
The description of the base recipe has been corrected as indicated.

masami256 commented:
thanks! merged.
